app/code/core/Mage/Adminhtml/Block/Notification/Window.php

```diff
@@ -144,7 +144,7 @@ public function getSeverityIconsUrl()
         if (is_null($this->_severityIconsUrl)) {
             $this->_severityIconsUrl =
                 (Mage::app()->getFrontController()->getRequest()->isSecure() ? 'https://' : 'http://')
-                . sprintf(Mage::getStoreConfig(self::XML_SEVERITY_ICONS_URL_PATH), Mage::getVersion(),
+                . sprintf((string)Mage::getStoreConfig(self::XML_SEVERITY_ICONS_URL_PATH), Mage::getVersion(),
```

Comment:
Note to self: This is not going to be needed if #2633  is merged.

Reply:
@elidrissidev #2633 is merged now :-)

Reply:
Done